TCP_Server.py

```diff
@@ -97,8 +97,11 @@ def close_connection(self, conn):
             return
         peername = self.current_peers[conn.get().fileno()]
         logging.info('closing connection to {0}'.format(peername))
-        del self.current_peers[conn.get().fileno()]
-        self.selector.unregister(conn.get())
+        try:
+            del self.current_peers[conn.get().fileno()]
+            self.selector.unregister(conn.get())
+        except KeyError:
+            logging.info('no peer...')
```

Comment:
You can always check first if peer is on list and then delete, but this is fine.
Just add which peer is missing to log.
"no peer" message says nothing :)

Reply:
You have right that `no peer...` message says nothing, but...
Look at line 99. Logic there write to log eg. `closing connection to 127.0.0.1:5490` and then if exception occures it will add `no peer...`.  All message will looks like:
```
closing connection to 127.0.0.1:5490
no peer...
```
Do you think there is a need to add `no 127.0.0.1:5490 peer...`?

Reply:
I didn't notice that. :)